### ISSUE TITLE
TBU1 Pads

### ISSUE THREAD

**Issue by yngndrw** [oxullo/uctester#2]
Hello,

According to the [datasheet](https://www.bourns.com/data/global/pdfs/TBU-CA.pdf) (Page 4), the TBU-CA protection device for thermal reasons. I'm not sure it's strictly necessary in this design, but it's best practice to follow the datasheet guidance where reasonable. They don't give specific dimensions, but their example appears to be approximately double the size of each of the conductor pads.

As an aside, do you think that the [TBU-CA085-100-WH](https://jlcpcb.com/partdetail/Bourns-TBU_CA085_100WH/C720638) would be a suitable replacement? I ask because it's the closest value which JLCPCB stock. The 85 means that it has an absolute maximum input voltage of 850V rather than 500V, but the 100 means that the trigger current is around 150mA rather than 75mA which I think should be fine - Worst-case R1 will dissipate around 5x the power it would have done with the TBU-CA050-050 but that's only for around 1uS.

**Comment by oxullo** [comment]
Hey @yngndrw,
I tried to dig into the matter of your aside-question in a post that you can find here: https://www.xul.it/blog/index.php/2022/10/03/power-cross-protection-in-the-uct/
TLDR: it should be ok :)

Regarding the footprint of the device, I followed the guidelines:

>  It is recommended that they should be the same dimension as the TBU® pads

(still page 4). As also mentioned there, increasing the pads would improve the thermal resistance, but given the triggering speed, quiescent and operating current of the device, I personally find it unnecessary.

One reason to extend the pads would be to slightly improve solderability, as it's quite complicated to see whether the pads have properly reflown when soldering by hand.
Besides, you can also see how Bourns itself didn't add any padding in their EVBs: https://www.bourns.com/docs/products-general/bourns_fu1106_rs-485_evalboard_designnote_1.pdf

I'll close the ticket, but please reopen if you disagree. In such case I'd be grateful if you'd provide a footprint. Thanks!
